File: EqualisePrices.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    int t;
    cin >> t;
    while (t--) {
        int n;
        cin >> n;
        vector<int> a(n);
        for (auto &x:a)
            cin >> x;
        int total=0;
        for (auto &x:a)
            total += x;
            int temp=total/n;
        if (total%n==0)
            cout << total/n << endl;
        else {
            while (n*temp<total)
                temp++;
            cout << temp << endl;
        }
    }

    return 0;
}

File: Python/EqualisePrices.py
def main():
    t = int(input())
    while t:
        t -= 1
        n = int(input())
        a = [int(input()) for i in range(n)]
        total = 0
        for i in a:
            total += i
        temp = total/n
        if total % n == 0:
            print(total/n)
        else:
            while n*temp < total:
                temp += 1
            print(temp)


if __name__ == '__main__':
    main()
